### DISCUSSION TITLE
SaltEdge Import problem

### DISCUSSION THREAD

**Discussion by vpedromarques** [firefly-iii/firefly-iii#5462] [Firefly III]
Hello! Happy New Year!
I've been struggling with Salt Edge implementation on the importer for some days. I already have an account on test mode. After requesting it to use with firefly-iii they finally gave me free access in order to include real banks. Created a service key and added it to the firefly-iii importer. It works and I can manage to select Portuguese banks. The problem is that every time I try to add a bank to be imported I keep getting "Server Error - QWAC is missing" and can't manage to associate any bank. Already sent a message to salt edge but had no luck. Anyone with this problem? How can I solve it?

Best regards!

**Comment by JC5** [comment]
If you have any logs from the importer that include this error and a stack trace, please let me know.

It seems it's a salt edge thing though.
